openhtf/io/test_record.py

```diff
@@ -71,3 +74,23 @@ class PhaseRecord(  # pylint: disable=too-few-public-methods,no-init
 
   See measurements.Record.GetValues() for more information.
   """
+
+
+class CodeInfo(mutablerecords.Record(
```

Comment:
Side note, there's a new requirement from my team to be able to disable sourcecode capturing on test phases. Not sure where the best place to do that check is, but wanted to get it out there.